### PR TITLE
Abc 1234/husky prepare commit msg hook

### PR DESCRIPTION
**Pull request by kibartas**

### CODE REVIEW

**Review by salmonela7 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,70 +1,16 @@
...
+## Frontend and husky setup:
+1. Run `yarn` at root directory
+2. Run `yarn` at frontend directory
+
+### Good job, you're done
+
+Backend setup
+1. You must have SQLEXPRESS installed.
+2. Make sure that your SQLEXPRESS connection string matches the one in appsettings.json.
+3. Open up CMD OR use Package manager console to run command "update-database".
```
Comment (tmaciulis22) [inline README.md]: lowercase or

File: README.md [hunk 2]
```diff
@@ -1,70 +1,16 @@
...
+4. Start the backend project.
+
+# To run the project
+1. Run `yarn start` at at frontend directory
```
Comment (tmaciulis22) [inline README.md]: `at at`

**Review by tmaciulis22 — APPROVED** [review]
Minor changes

**Review by AArlauskas — APPROVED** [review]
LGTM

**Pull request merged**